src/routes/savingsRoutes.ts

```
@@ -1,6 +1,6 @@
 import { Router, type IRouter } from "express";
 import { validateApiKey } from "../middleware/auth";
-import { requireSegmentScope } from "../middleware/segmentGuard";
+import { requireMinTier, requireSegmentScope } from "../middleware/segmentGuard";
```

ESLint

[error] Replace `·requireMinTier,·requireSegmentScope·` with `⏎··requireMinTier,⏎··requireSegmentScope,⏎`

(prettier/prettier)